tests/lax_scipy_sparse_test.py

```diff
@@ -98,36 +96,40 @@ def _fetch_preconditioner(self, preconditioner, A, rng=None):
     shape=[(4, 4), (7, 7)],
     dtype=[np.float64, np.complex128],
     preconditioner=[None, 'identity', 'exact', 'random'],
+    assume_ipart_is_zero=[False, True]
   )
-  def test_cg_against_scipy(self, shape, dtype, preconditioner):
+  def test_cg_against_scipy(self, shape, dtype, preconditioner,
+                            assume_ipart_is_zero):
     if not config.enable_x64.value:
       raise unittest.SkipTest("requires x64 mode")
 
     rng = jtu.rand_default(self.rng())
     A = rand_sym_pos_def(rng, shape, dtype)
+    if assume_ipart_is_zero:
+      A = A - jnp.imag(A)
     b = rng(shape[:1], dtype)
     M = self._fetch_preconditioner(preconditioner, A, rng=rng)
 
     def args_maker():
       return A, b
 
     self._CheckAgainstNumpy(
-        partial(scipy_cg, M=M, maxiter=1),
-        partial(lax_cg, M=M, maxiter=1),
-        args_maker,
-        tol=1e-12)
+      partial(scipy_cg, M=M, maxiter=1),
+      partial(lax_cg, M=M, maxiter=1,assume_ipart_is_zero=assume_ipart_is_zero),
+      args_maker,
+      tol=1e-12)
```

Comment:
Please avoid unrelated formatting changes.

Reply:
This is to prevent going over 80 char

Reply:
JAX doesn't require lines to be within 80 characters, and formatting changes unrelated to the primary purpose of the PR make it more difficult to review. Please revert unrelated changes.